DEVELOPMENT.md

```diff
@@ -1,78 +1,92 @@
 # Development Setup
 
-This guide outlines the steps to set up a development environment for Dojo. If you just want to play with the toolchain, we recommend following the [Quick Start](/getting-started.md) guide instead.
+This guide outlines the steps for setting up an environment for developing the Dojo codebase.
+If you want to use Dojo to make things, follow the [Dojo Installation guide](https://book.dojoengine.org/installation) instead.
 
-## Prerequisites
+## System prerequisites
 
 - [Rust](https://github.com/rust-lang/rust)
 - [Cairo](https://github.com/starkware-libs/cairo)
+- [Katana](https://github.com/dojoengine/katana) (via Dojo)
 - [protoc](https://github.com/protocolbuffers/protobuf)
+- Optional: [VS Code extension](https://marketplace.visualstudio.com/items?itemName=starkware.cairo1) for syntax highlighting
 
-## Setup
+See the [Dojo Installation guide](https://book.dojoengine.org/installation) for more details on installing these dependencies.
+
+> Depending on your Linux distribution, you may need to install additional dependencies.
+
+## Setting up your environment
 
 ### 1. Clone the repository
 
 ```sh
-git clone https://github.com/dojoengine/dojo.git
+# Clone and enter the repo
+git clone https://github.com/dojoengine/dojo.git && cd dojo
 ```
 
-### 2. Install Rust and dependencies
-
-Install & update Rust:
+### 2. Run the tests
 
-```sh
-rustup override set stable && rustup update
-```
+We're using `nextest` as our test runner, which you can get at [https://nexte.st/](https://nexte.st/).
 
-Now run the test suite to confirm your setup:
+> Note that the tests depend on database artifacts for a simple "spawn and move" game.
 
 ```sh
-# First generate db artifacts for the tests
+# Prepare the spawn-and-move db artifact
 bash scripts/extract_test_db.sh
 
-# The run the tests
-cargo test
-```
+# Run all the tests
+cargo nextest run --all-features --workspace
 
-Note: Depending on your Linux distribution, you may need to install additional dependencies.
+# Run a single package test
+cargo nextest run --all-features -p sozo-ops
+```
 
-### 3. Install Scarb package manager
+By convention, Dojo stores test dependencies in your system's `/tmp/` directory.
```

Comment:
This applies to the Katana runner path in the scripts + the database location yeah. Maybe we could add the list/naming them to not be too generic? WDYT?